WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsViewModel.swift

```diff
@@ -246,12 +256,8 @@ final class WooShippingCreateLabelsViewModel: ObservableObject {
                 }
             }
 
-            let totalOrderItems = order.items.map(\.quantity).reduce(0, +)
-            if totalOrderItems > 1 {
-                // Only fetch shipments info if there are more than one order items.
-                group.addTask {
-                    await self.loadShipmentsInfo()
-                }
+            group.addTask {
+                await self.loadShipmentsInfo()
```

Comment:
Previously, we wanted to fetch shipment info only when there is more than one order item because this info is used only to build shipments. Splitting shipments is not supported when there is one order item, so shipment info is unnecessary.

Now since we need shipment info to determine whether a shipment's label has been purchased, fetching the info is always necessary.